src/main/scala/mesosphere/marathon/api/akkahttp/Rejections.scala

```diff
@@ -20,16 +20,16 @@ object Rejections {
     private def readableVersion(version: Option[Timestamp]) = version.fold("")(v => s" in version $v")
 
     def noApp(id: PathId, version: Option[Timestamp] = None): EntityNotFound = {
-      EntityNotFound(Message(s"App '$id' does not exist ${readableVersion(version)}"))
+      EntityNotFound(Message(s"App '$id' does not exist${readableVersion(version)}"))
```

Comment:
why was the space removed (here and below)?

Reply:
This is confusing, I know. So `readableVersion(version)` already has a space prepended. We would end up with two. If `readableVersion(version)` is empty we would have an extra space.